console/src/app/modules/form-field/field/form-field.component.html

```diff
@@ -7,7 +7,7 @@
   </label>
 </ng-template>
 
-<div class="cnsl-form-field-wrapper" (click)="_control.onContainerClick && _control.onContainerClick($event)">
+<div class="cnsl-form-field-wrapper" (click)="_control.onContainerClick($event)">
```

Comment:
Is this never undefined anymore?

Reply:
nope.. it never was, but the linter reacted differently now